src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,54 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { TabsInterface } from '../../types/tabs';
+
+type Props = {
+  tabs: TabsInterface[];
+  selectedTabId: string;
+  onTabSelected: (tab: TabsInterface) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(({ id }) => id === selectedTabId) || tabs[0];
+
+  return (
+    <>
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map((tab) => {
+              return (
+                <li
+                  className={classNames({
+                    'is-active': selectedTabId === tab.id,
```

Comment:
```suggestion
                    'is-active': selectedTab.id === tab.id,
```
Probably that's the reason why you can't pass the last test
![image](https://user-images.githubusercontent.com/96409563/200778182-f5f8457f-6a4f-49ec-9db2-ba4ca873f847.png)